lib/LoginStrategy.ts

```diff
@@ -266,8 +272,13 @@ export class LoginStrategy extends BaseStrategy {
         this.cacheServerClient?.address === userAddress
           ? []
           : await this.credentialResolver.eip191JwtsOf(userDid);
+      const roleClaims = this.includeAllRoles
+        ? userClaims
+        : userClaims.filter((claim) =>
+            this.acceptedRoles.has(claim.payload.claimData.claimType)
```

Comment:
I think we should check that `claimType` is available in the payload to available null-reference error. Even if we're sure of the behaviour of `eip191JwtsOf` currently (i.e. we know that it always return JWTs with `claimType`) I think it's good to have this check in case the behaviour changes.
```suggestion
        : userClaims.filter((claim) => {
            const claimType = claim?.payload?.claimData?.claimType;
            return claimType && this.acceptedRoles.has(claimType);
          }
```